### PR TITLE
Minor grammar changes for README

### PR DESCRIPTION
**Pull request by R167**
Found a few minor edits you might want to consider on grammar to make it read a bit clearer. Also, one spot where I really wasn't understanding what was going on (highlighted in diff view)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ If you're already using a pre-forking web server, you'll need to do most of the
 
 ## Pre-Booting
 
-If your application can't be pre-forked, you can at load the gem files and libraries your application depends instead by specifying a preboot file. This file will be loaded by the master process and any code loaded will be copy-on-write shared with the child processes.
+If your application can't be pre-forked, you can at load the gem files and libraries your application depends on instead by specifying a preboot file. This file will be loaded by the master process and any code loaded will be copy-on-write shared with the child processes.
```
Comment (R167) [inline README.md:40]: I'm not really sure what you're saying here. Clarify???

**Comment by R167** [comment]
For handling the option parsing, I would generally think to go with something well defined like `optparse` rather than the homegrown way you're pulling out the options. For example:

```ruby
#!/usr/bin/env ruby
# frozen_string_literal: true

require 'optparse'
require_relative "../lib/sidekiq-process_manager"

DEFAULT_PROCESS_COUNT = 1

options = {
  process_count: [1, Integer(ENV.fetch('SIDEKIQ_PROCESSES', DEFAULT_PROCESS_COUNT))].max,
  prefork: !ENV.fetch("SIDEKIQ_PREFORK", "").empty?,
  preboot: ENV["SIDEKIQ_PREBOOT"],
  mode: nil,
}

OptionParser.new do |opts|
  opts.banner = "Usage: sidekiq-process-manager [options] [[--] sidekiq options]"

  opts.on('--processes PROCESSES', Integer, "Number of processes to spin up") do |count|
    options[:process_count] = [count, 1].max
  end

  opts.on('--[no-]prefork', "Use prefork for spinning up sidekiq processes") do |prefork|
    options[:prefork] = prefork
  end

  opts.on('--preboot FILE', "Preboot the processes") do |preboot|
    options[:preboot] = preboot
  end

  opts.on('--testing', "Enable test mode") do |testing|
    options[:mode] = :testing if testing
  end

  opts.on("--help", "Prints this help") do
    puts opts
    exit
  end

  opts.separator(<<~DESCR)

  After the manager options, pass in any options for the sidekiq processes.
  Additionally, passing in the optional `--` will explicitly end the manager options and begin the sidekiq opts.
  E.g.
      $ sidekiq-process-manager --no-prefork -- -C config/sidekiq.rb
      Calls sidekiq with `sidekiq -C config/sidekiq.rb`
  DESCR
end.parse!

sidekiq_args = []
begin
  parser.order!(ARGV) { |nonopt| sidekiq_args << nonopt }
rescue OptionParser::InvalidOption => err
  # Handle the case where a user doesn't put in the `--` to separate the args
  sidekiq_args.concat(err.args)
end

ARGV[0, 0] = sidekiq_args

### LOG PARAMS
p options
p ARGV

begin
  manager = Sidekiq::ProcessManager::Manager.new(**options)
  manager.start
rescue => e
  STDERR.puts e.message
  STDERR.puts e.backtrace.join($/)
  exit 1
end
```

File: README.md (hunk 1 above)
Comment (bdurand) [reply README.md:40]: The difference between pre-forking and pre-booting is pre-forking loads the application code before forking while pre-booting only loads the gems.

**Comment by bdurand** [comment]
I'd started using option parser, but I didn't want to impact the args sent to sidekiq. Looks like you solved that, though. Can you open a PR with the option parser code in it.

**Comment by R167** [comment]
Makes sense to me. Proof of concept code was working pretty good when i was using it, so will open another PR.

**Pull request merged**